### ISSUE TITLE
private_contact_point and public_contact_point are null

### ISSUE THREAD

**Issue by cameronattard** [instaclustr/terraform-provider-instaclustr#67]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

v0.12.x

### Affected Resource(s)

* instaclustr_cluster

### Terraform Configuration Files

```hcl

variable "cassandra_node_size" {
  default = "m5l-250-v2"
}

variable "cassandra_sla_tier" {
  default = "NON_PRODUCTION"
}

variable "cassandra_private_network_cluster" {
  default = false
}

variable "cassandra_number_of_racks" {
  default = 3
}

variable "cassandra_nodes_per_rack" {
  default = 1
}

variable "cassandra_version" {
  default = "3.11.8"
}

resource "instaclustr_cluster" "cassandra" {
  cluster_name            = var.cluster_name
  node_size               = var.cassandra_node_size
  data_centre             = replace(upper(var.aws_region), "-", "_")
  sla_tier                = var.cassandra_sla_tier
  cluster_network         = var.cassandra_cluster_network
  private_network_cluster = var.cassandra_private_network_cluster
  cluster_provider = {
    name                = "AWS_VPC"
    account_name        = var.cassandra_cluster_provider_account_name
    disk_encryption_key = var.encryption_key_id
  }
  rack_allocation = {
    number_of_racks = var.cassandra_number_of_racks
    nodes_per_rack  = var.cassandra_nodes_per_rack
  }
  bundle {
    bundle  = "APACHE_CASSANDRA"
    version = var.cassandra_version
    options = {
      auth_n_authz                      = true
      client_encryption                 = true
      use_private_broadcast_rpc_address = true
      lucene_enabled                    = true
    }
  }
}

```

### Expected Behavior

The following two outputs should have correct values:

```hcl
# NOTE: this currently returns null for some reason...
output "cassandra_public_contact_point" {
  value = instaclustr_cluster.cassandra.public_contact_point
}

# NOTE: this currently returns null for some reason...
output "cassandra_private_contact_point" {
  value = instaclustr_cluster.cassandra.private_contact_point
}
```

### Actual Behavior

Both outputs are null.

### Steps to Reproduce

1. `terraform apply`

**Comment by peter-toms** [comment]
Thanks @cameronattard appreciate the info INS-12901 added to our internal task tracking system for prioritisation by the team, Cheers!

**Comment by peter-toms** [comment]
An update on this item, we are nearing completion and expect to have the feature ready early next week (AEST).  Thanks again for the patience, will keep you updated on our progress.

**Comment by cameronattard** [comment]
Thanks for the update, @peter-toms!

**Comment by Chrismelba-Instaclustr** [comment]
Hi there. This should be fixed in release 1.9.6! Thanks so much for bringing it to our attention.

**Comment by Chrismelba-Instaclustr** [comment]
Just a quick note. For this to work as intended you'll need to add "wait_for_state="RUNNING"" to the clusters you wish to get the contact points for, as these IPs aren't known until the cluster is running
